### ISSUE TITLE
ProjectTopic's  __str__()

### ISSUE THREAD

**Issue by raphaelyancey** [Ircam-Web/mezzanine-organization#1]
Line 130, is the *parent name - topic key* form voluntary? Shouldn't it be more consistent like *parent name - topic name* or *parent key - topic key* ?

https://github.com/Ircam-Web/mezzanine-organization/blob/f55791492f35f4a4d923a095400ce04674f050e8/organization/projects/models.py#L118-L132

**Comment by yomguy** [comment]
You are right, the current schema is not recursively consistent. Let's use *object.name* only.
